src/core/sys/windows/dll.d

```diff
@@ -328,7 +328,9 @@ public:
  */
 bool dll_fixTLS( HINSTANCE hInstance, void* tlsstart, void* tlsend, void* tls_callbacks_a, int* tlsindex ) nothrow
 {
-    version (Win64)
+    version (GNU_EMUTLS)
+        return true;
```

Comment:
Unless this is in upstream, add a comment at the top of the file that it has been modified from original upstream.